src/Microsoft.ML.HalLearners/SymSgdClassificationTrainer.cs

```diff
@@ -754,6 +754,9 @@ private void CheckLabel(RoleMappedData examples, out int weightSetCount)
 
         private static unsafe class Native
         {
+            //To triger the loading of MKL library since MKL proxy native library depends on it.
```

Comment:
`MKL proxy` isn't used by SymSGD, but instead `SymSGDNative` is.

Reply:
Yikes, copy paste habits ...

---
In reply to: [222123162](https://github.com/dotnet/machinelearning/pull/977#discussion_r222123162) [](ancestors = 222123162)